packages/wonder-blocks-timing/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "@khanacademy/wonder-blocks-timing",
+    "version": "0.0.1",
+    "design": "v1",
+    "publishConfig": {
+      "access": "public"
+    },
+    "description": "",
+    "main": "dist/index.js",
+    "module": "dist/es/index.js",
+    "source": "index.js",
+    "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "peerDependencies": {
+      "react": "^16.4.1"
```

Comment:
Doesn't seem like this is needed?

Reply:
It will be once this package is finished.